src/index.ts

```diff
@@ -67,18 +94,19 @@ export const useCourseSearch = (
   clearSearch: () => void,
   aggregations: Aggregations,
   loaded: boolean,
-  searchPageSize: number
-) => {
+  searchPageSize: number,
+  history: HHistory
```

Comment:
This is the main change.

Previously course-search-utils was using its own history object, which meant it could not be properly integrated with react router.

⚠️ This is a breaking change. I originally was going to make the `history` param optional, but I think that's basically leaving a trap for us in the future. E.g., OCW uses `useCourseSearch` and has no router, so it would be fine at the moment. But if we ever added a router to OCW search, we would have the same problem we currently have in infinite corridor.